object.c

```diff
@@ -1998,14 +1998,26 @@ rb_mod_gt(VALUE mod, VALUE arg)
 
 /*
  *  call-seq:
- *     module <=> other_module   -> -1, 0, +1, or nil
+ *     self <=> object -> -1, 0, +1, or nil
  *
- *  Comparison---Returns -1, 0, +1 or nil depending on whether +module+
- *  includes +other_module+, they are the same, or if +module+ is included by
- *  +other_module+.
+ *  Returns:
+ *
+ *  - +-1+, if +self+ includes +object+, if or +self+ is derived from +object+.
```

Comment:
Instead of "derived", can we just use "subclass" or "superclass" instead? AFAIK "derived" isn't any standard terminology in Ruby.

Reply:
Fixed.